app/frontend/src/features/profile/view/Overview.tsx

```diff
@@ -53,9 +64,18 @@ export default function Overview({ user }: OverviewProps) {
   return (
     <Card className={classes.card}>
       <Avatar {...{ user }} className={classes.grow} />
-      <Typography variant="h1">{user.name}</Typography>
-      <Typography variant="body1">{user.city}</Typography>
+      <Typography variant="h1" className={classes.intro}>
+        {user.name}
+      </Typography>
+      <Typography variant="body1" className={classes.intro}>
+        {user.city}
+      </Typography>
       <Divider />
+      <CardActions className={classes.cardActions}>
```

Comment:
This is shown on all profiles at the moment. Need some additional to check that the user ID matches the current user's before showing it.